pubsub.go

```diff
@@ -183,6 +183,17 @@ type PubSubRouter interface {
 	Leave(topic string)
 }
 
+type AcceptStatus int
+
+const (
+	// AcceptAll signals to accept the incoming RPC for full processing
+	AcceptNone AcceptStatus = iota
+	// AcceptControl signals to accept the incoming RPC only for control message processing
```

Comment:
`control message processing` can mean a lot of things. We should explain this in godocs. Either here, or elsewhere. If elsewhere, link to it.